lib/resty/etcd/v3.lua

```diff
@@ -239,19 +243,26 @@ function _M.new(opts)
         })
     end
 
+    local sema, err = semaphore.new()
+    if not sema then
+        return nil, err
+    end
+
     if opts.health_check then
         return setmetatable({
-            last_auth_time   = now(), -- save last Authentication time
-            jwt_token        = nil,       -- last Authentication token
-            is_auth          = not not (user and password),
-            user             = user,
-            password         = password,
-            timeout          = timeout,
-            ttl              = ttl,
-            is_cluster       = #endpoints > 1,
-            endpoints        = endpoints,
-            key_prefix       = key_prefix,
-            ssl_verify       = ssl_verify,
+            last_auth_time = now(), -- save last Authentication time
```

Comment:
we'd better keep the old style, only update the code we need to update

![image](https://user-images.githubusercontent.com/6814606/101567999-cbe91200-3a0c-11eb-9e23-d01a326f7fd2.png)

Reply:
got, recover it